plugins/mimecast/help.md

```diff
@@ -1017,7 +1017,7 @@ Most common cloud [URLs](https://www.mimecast.com/tech-connect/documentation/api
 
 # Version History
 
-* 5.3.7 - Task `monitor_siem_logs` adding in sanitisation for names of files to be read in.
+* 5.3.7 - Task `monitor_siem_logs` adding in sanitization for names of files to be read in | implement SDK 5.4 to include new task custom_config | bump setuptools.
```

Comment:
Suggested reword: Include SDK 5.4 which adds new task custom_config parameter

Reply:
Thanks, updated this now.